### PR TITLE
Prevent readonly models in relation generator

### PR DESCRIPTION
**Pull request by jannyHou**
This pr is created for preventing users from creating a relation on a read-only model

Connect to strongloop/generator-loopback#103

### CODE REVIEW

**Comment by jannyHou** [comment]
hey @bajtos I am trying to solve the problem in https://github.com/strongloop/generator-loopback/issues/103 by providing readonly Models in prompt choices.
Do you have any ideas on this solution? Thanks!

And if it works for relation generator, I think the same choices should apply to acl and model generators.
I haven't check other generators carefully and this is just a thought, because my strongloop suddenly crushed(I got "Error loading module loopback" with `slc loopback`), so sorry for that! I am trying my best to fix my local strongloop.

File: lib/actions.js [hunk 1]
```diff
@@ -104,6 +107,12 @@ actions.loadModels = function() {
     this.modelNames = results.map(function(m) {
       return m.name;
     });
+    this.editEnableModels = results.filter(function(result) {
```
Comment (bajtos) [inline lib/actions.js]: I find it difficult to understand the name `editEnableModel`, how about `editableModels` or `writeableModels`? Similarly for the list of model names below.

**Comment by bajtos** [comment]
Hi @jannyHou, this is an elegant solution! 

I wish we had infrastructure in place that would allow us to write an automatic test for changes like these. Never mind, let's leave that out of scope of this PR.

> I think the same choices should apply to acl and model generators.

ACL generator - yes, that makes sense.
Mode generator is creating a new generator, which is by definition editable/writable, right?

Would you like to fix the ACL generator as part of this PR, since the change should be pretty small?

**Comment by jannyHou** [comment]
hey @bajtos ,thank you! And sure I will update acl generator as well.

File: lib/actions.js (hunk 1 above)
Comment (jannyHou) [inline lib/actions.js]: sure `editable` sounds perfect!

**Comment by bajtos** [comment]
Great stuff, the patch LGTM now.

@jannyHou Please squash the commits to a single one and edit the description to mention all three affected generators. Make sure the first line (commit summary) fits into 50 characters, see the [50/72 rule](http://tbaggery.com/2008/04/19/a-note-about-git-commit-messages.html). You can land the patch yourself then, no need for any further reviews.

**Pull request merged**